apps/home/models.py

```diff
@@ -31,6 +31,13 @@ class HomePage(Page):
     subtitle_de_ls = models.CharField(
         max_length=120, blank=True, verbose_name=('Home page title de ls'))
 
+    headertext_de = models.CharField(
+        max_length=255, blank=True, verbose_name=('Home page subtitle de'))
+    headertext_en = models.CharField(
+        max_length=255, blank=True, verbose_name=('Home page subtitle en'))
+    headertext_de_ls = models.CharField(
+        max_length=255, blank=True, verbose_name=('Home page subtitle de ls'))
+
```

Comment:
I would prefer to change `subtitle` to `headertitle` and name this `headersubtitle` but as we use subtitle everywhere I wasn't sure if it was worth it?

Reply:
I am fine with renaming it!

Reply:
coolio :)